x-pack/plugins/fleet/server/routes/package_policy/handlers.test.ts

```diff
@@ -300,9 +360,13 @@ describe('When calling package policy', () => {
       };
       const request = getUpdateKibanaRequest(newData as any);
       await routeHandler(context, request, response);
+      const responseItem = { ...existingPolicy, ...newData };
       expect(response.ok).toHaveBeenCalledWith({
-        body: { item: newData },
+        body: { item: responseItem },
       });
+
+      const validationResp = PackagePolicyResponseSchema.validate(responseItem);
```

Comment:
this way I'm making sure the response schema works, trying to fill all fields in objects, including optional ones